code/content.js

```diff
@@ -1,17 +1,38 @@
-// Wait for the first part of the page to load
-$(document).ready(function () {
+var calendar_grid_selector = 'div[role="grid"]';
+var body = document.querySelector('body');
+var calendar_grid = document.querySelectorAll(calendar_grid_selector);
 
-    // Wait for the rest of the fancy dynamic stuff to load.
-    // 5 seconds should be enough.
-    setTimeout(function () {
-
-        // Get a handle on the calendar grid
-        $('div[role="grid"]').on('mousewheel', function (e) {
-            
-            // Scrolling.... hahhahahaha I don't think so
+var disable_scroll = function () {
+    for (var live_selector of document.querySelectorAll(calendar_grid_selector)) {
+        live_selector.addEventListener('mousewheel', function (e) {
             if (e.target.id == 'el') return;
```

Comment:
Is this line still needed? I could not find any purpose for this, maybe this was needed on the previous Calendar version?